serverless/src/utils/slack-processor-util.ts

```diff
@@ -0,0 +1,84 @@
+import * as uuid from "uuid";
+import {WebClient} from '@slack/web-api';
+
+// Configure client
+const client = new WebClient(process.env.SLACK_OAUTH_TOKEN);
+const supportChannel: string = "support-channel";
+
+/**
+ * Create session chat with generated UUID
+ */
+export async function createSessionChannel(message: string): Promise<string> {
+    const id: string = uuid.v4();
+
+    await client.conversations.create({
+        token: process.env.SLACK_OAUTH_TOKEN,
+        name: uuid.v4(),
```

Comment:
what does this name field do?

Reply:
this creates a slack channel, with its name being the uuid of the session.